tls/s2n_cipher_suites.c

```diff
@@ -834,8 +879,38 @@ static int s2n_wire_ciphers_contain(const uint8_t * match, const uint8_t * wire,
     return 0;
 }
 
+static int s2n_cipher_is_compatible_with_cert(struct s2n_cipher_suite *cipher, struct s2n_cert *cert, uint8_t *compatibility_out)
+{
+    *compatibility_out = 0;
+
+    notnull_check(cert);
```

Comment:
nit: I don't think this is adding too much value since this is a static helper and we notnull checked cert right before calling this helper